js/foundation.slider.js

```diff
@@ -106,7 +106,11 @@ class Slider {
    * @fires Slider#changed
    */
   _setHandlePos($hndl, location, noInvert, cb) {
-  //might need to alter that slightly for bars that will have odd number selections.
+    // don't move if the slider has been disabled since its initialization
+    if (this.$element.hasClass(this.options.disabledClass)) {
```

Comment:
I'd prefer there be a single source of truth, and it looks like we're currently using the class as the way to do that.  @ncoden did you see someplace in particular that is setting data-disabled?  I'm not seeing that mechanism being used.